highlight.io/components/QuickstartContent/self-host/shared-snippets.tsx

```diff
@@ -1,4 +1,3 @@
-import { siteUrl } from '../../../utils/urls'
```

Comment:
This was an unused import that my editor clean up. Would be nice to add some eslint rules around that like we have in the frontend app:

https://github.com/highlight/highlight/blob/ebb0f1e751caa97b0a3399cfd2fc9615e196e916/frontend/.eslintrc#L46

Reply:
could we share the `.eslintrc` among all packages in the monorepo?

Reply:
Probably yes. Would likely require some clean up. I've added that as a hack week idea (#4862).